include/gbinder_local_object.h

```diff
@@ -61,6 +61,15 @@ GBinderLocalReply*
 gbinder_local_object_new_reply(
     GBinderLocalObject* obj);
 
+GBINDER_STABILITY_LEVEL
+gbinder_local_object_get_stability(
+    GBinderLocalObject* self);
```

Comment:
Please add `/* Since 1.0.40 */` comment (here and in the .c file) so that the person reading this code knows which version of libgbinder introduced this API (and therefore which libgbinder version the program would require if it uses this API).